adapters/repos/db/inverted/prop_length_tracker.go

```diff
@@ -247,6 +247,33 @@ func (t *PropertyLengthTracker) PropertyMean(propName string) (float32, error) {
 	return sum / totalCount, nil
 }
 
+// BM25F
+func (t *PropertyLengthTracker) PropertyCount(propName string) (float32, float32, error) {
```

Comment:
this is almost the same as PropertyMean. would be nice to extract and reuse common parts